src/test/java/com/snowflake/kafka/connector/ConnectClusterBaseIT.java

```diff
@@ -75,33 +66,19 @@ public void after() {
     SnowflakeFakeSinkTask.resetRecords();
   }
 
-  @Test
-  public void connectorShouldConsumeMessagesFromTopic() {
-    connectCluster.kafka().produce(TEST_TOPIC, "test1");
-    connectCluster.kafka().produce(TEST_TOPIC, "test2");
-
-    await()
-        .untilAsserted(
-            () -> {
-              List<SinkRecord> records = SnowflakeFakeSinkTask.getRecords();
-              assertThat(records).hasSize(2);
-              assertThat(records.stream().map(SinkRecord::value)).containsExactly("test1", "test2");
-            });
-  }
-
-  protected Map<String, String> createProperties() {
+  final Map<String, String> defaultProperties(String topicName, String connectorName) {
     Map<String, String> config = new HashMap<>();
 
     // kafka connect specific
     // real connector will be specified with SNOW-1055561
     config.put(CONNECTOR_CLASS_CONFIG, SnowflakeFakeSinkConnector.class.getName());
```

Comment:
As the comment reads, we should use a real connector implementation here instead of `SnowflakeFakeSinkConnector`.
So that we can verify the whole flow of _a real_ implementation.

Reply:
I replaced it with the real one, could you please check that?

Reply:
Are you planning to remove `SnowflakeFakeSinkConnector` in another PR?
We don't need to have once we have a real test.

Reply:
removed, the mentioned class and `FakeSinkTask`